server/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -387,6 +387,9 @@ private LiTag renderPublishModalProgramItem(
       ProgramDefinition program, ImmutableList<Long> universalQuestionIds) {
     String visibilityText = " ";
     switch (program.displayMode()) {
+      case DISABLED:
+        visibilityText = " (Hidden from applicants and TIs) ";
```

Comment:
suggestion (non-blocking): I think we have a preference for writing out "Trusted Intermediaries" since people might be unfamiliar with the "TI" acronym.

Reply:
sounds good, changed it to "Trusted Intermediaries"